kernel/delta/io.rb

```diff
@@ -0,0 +1,34 @@
+# Redefine STDIN, STDOUT & STDERR to use the new IO class. It reopened and redefined
+# all methods used in the bootstrap step. Secondly, update the $std* globals to point
+# to the new objects.
+
+def redefine_io(fd, mode)
```

Comment:
This method should eventually most likely be in a mirror, or scoped somewhere under the `Rubinius` namespace in general. Right now it's defined at the top-level, which is likely to clash with other methods.